code/game/machinery/doors/brigdoors.dm

```diff
@@ -238,63 +290,63 @@
 /obj/machinery/door_timer/attack_ghost(mob/user)
 	interact(user)
 
-//Allows humans to use door_timer
-//Opens dialog window when someone clicks on door timer
-// Allows altering timer and the timing boolean.
-// Flasher activation limited to 150 seconds
 /obj/machinery/door_timer/attack_hand(mob/user)
 	if(..())
 		return
 	interact(user)
 
 /obj/machinery/door_timer/interact(mob/user)
-	// Used for the 'time left' display
+	//Used for the 'time left' display
 	var/second = round(timeleft() % 60)
 	var/minute = round((timeleft() - second) / 60)
-
-	// Used for 'set timer'
+	//Used for 'set timer'
 	var/setsecond = round((timetoset / 10) % 60)
 	var/setminute = round(((timetoset / 10) - setsecond) / 60)
-
 	user.set_machine(src)
 
-	// dat
-	var/dat = "<HR>Timer System:</hr>"
-	dat += " <b>Door [id] controls</b><br/>"
+	var/dat = {"<center><h2>Timer System:<br>
+		<b>Door [id]</b></h2></center><hr>"}
 
-	// Start/Stop timer
-	if(timing)
-		dat += "<a href='?src=[UID()];timing=0'>Stop Timer and open door</a><br/>"
-	else
-		dat += "<a href='?src=[UID()];timing=1'>Activate Timer and close door</a><br/>"
+	if(screen)//Main Menu
+		dat += {"<br><b>Detainee Name:	</b><i>[detaineename]</i>			<a href='?src=[UID()];pickname=1'>Set Name</a>
+				<br><b>Charged with violation of:	</b><br><i>[detaineecrimes]</i><br>

```

Comment:
If I'm correct in my thinking then this will output as "Charged with violation of: murder." for example which doesn't make sense. It should be something along the lines of "Charged with: murder."